src/utils/errors.rs

```diff
@@ -85,7 +85,7 @@ impl ApiError {
         )
     }
 
-    fn new(status_code: u16, message: ErrorDetails) -> Self {
+    pub fn new(status_code: u16, message: ErrorDetails) -> Self {
```

Comment:
This is because of tests?

Reply:
No, is because of this line: https://github.com/gnosis/safe-client-gateway/blob/48138f5c7f3ef757747fb0fb6dfac1ab6164dacb/src/cache/cache_op_executors.rs#L69